ui/app/services/keyboard.js

```diff
@@ -417,7 +418,7 @@ export default class KeyboardService extends Service {
             command.label === 'Hide Keyboard Shortcuts'
           ) {
             event.preventDefault();
-            command.action();
+            command.action(command.element);
```

Comment:
Side effect: At https://github.com/hashicorp/nomad/pull/19084/files#diff-584468c2e8e339d3e2cf7238906d722689a177a0ab7c0d92f7fbd795262bb819R19-R24 I wanted to do some "expand dropdown" functionality when the user typed `a c`, but there was no idiomatic way to do this with the design system component in question, so I needed to get the element passed along to traverse to its child and click its button. Made me miss jQuery a bit, if I'm being honest.